packages/cli/src/executions/execution.service.ts

```diff
@@ -478,4 +483,17 @@ export class ExecutionService {
 
 		return await this.executionRepository.stopDuringRun(execution);
 	}
+
+	async enrichExecutionSummariesWithScopes(user: User, summaries: ExecutionSummary[]) {
+		const summariesWithScopes = summaries as Array<ExecutionSummary & { scopes: Scope[] }>;
```

Comment:
Instead of overriding the arg type, what about a new type `ScopedExecutionSummary`?